docs/installation/installation-using-docker-linux.md

```diff
@@ -1,5 +1,9 @@
 # Installation Using Docker for Linux
 
+This guide covers building a new projects based on Shopsys Framework.
+Maybe you want to contribute to the framework itself.
+In that case don't forget to take a look at an article about [Monorepo](../introduction/monorepo.md).
```

Comment:
```suggestion
you need to install the whole [shopsys/shopsys](https://github.com/shopsys/shopsys) monorepo. Take a look at an article about [Monorepo](../introduction/monorepo.md) for more information.
```

Reply:
if you do agree with my suggestion, please perform these changes in all the others installation guides.